SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security Policy
+
+## Threat Model
+
+List assumptions (rez will run inside an internal network, we assume good intentions, package definitions are written in python and executed)
+
+Access to shared filesystem
+
+The current assumptions are:
+* Rez was designed to be used within a studio environment.
+* Package definitions, both for building packages and resulting from a build are Python files (`package.py`). Rez will read and load them in memory at resolve time.
+* Rez config files can be written in YAML or Python.
+* Package definitions and config files written in Python can contain arbitrary code.
+* Rez will create new shells via subprocesses.
```

Comment:
Should we state that shells can be nested? I feel like we should since that's differentiator from other environment managers (conda, virtualenv, etc).